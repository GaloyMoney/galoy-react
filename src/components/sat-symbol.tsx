export const SatSymbol = () => (
  <i aria-hidden className="fak fa-satoshisymbol-solidtilt sat-symbol" />
)
