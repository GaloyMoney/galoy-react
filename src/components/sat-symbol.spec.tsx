import { render } from "@testing-library/react"

import { SatSymbol } from "./sat-symbol"

describe("SatSymbol", () => {
  it("renders and matches snapshot", () => {
    const { asFragment } = render(<SatSymbol />)
    expect(asFragment()).toMatchSnapshot()
  })
})
