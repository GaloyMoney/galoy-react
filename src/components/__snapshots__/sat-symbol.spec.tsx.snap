// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SatSymbol renders and matches snapshot 1`] = `
<DocumentFragment>
  <i
    aria-hidden="true"
    class="fak fa-satoshisymbol-solidtilt sat-symbol"
  />
</DocumentFragment>
`;
